shell/platform/tizen/flutter_tizen_view.cc

```diff
@@ -252,7 +260,12 @@ void FlutterTizenView::OnCommit(const std::string& str) {
 }
 
 void FlutterTizenView::SendInitialGeometry() {
-  OnRotate(window_->GetRotation());
+  if (tizen_view_->GetType() == "window") {
+    auto window = reinterpret_cast<TizenWindow*>(tizen_view_.get());
+    OnRotate(window->GetRotation());
+  } else {
+    OnRotate(0);
```

Comment:
Are there any plans to connect the orientation of the application and the orientation of the view via Embedder APIs? It doesn't seem appropriate to always leave it at 0. 
If it's unfinished feature, you can add the TODO comment or use `FT_UNIMPLEMENTED` macro.

Reply:
I needed `SendWindowMetrics(..)`. updated the code